tests/sentry/api/endpoints/test_organization_member_team_details.py

```diff
@@ -489,15 +490,16 @@ def test_admin_cannot_remove_member(self):
     def test_admin_cannot_remove_member_using_user_token(self):
         # admin not in team
         self.login_as(self.admin)
-        token = self.create_user_auth_token(user=self.admin_user, scope_list=["team:write"])
-        self.get_error_response(
+        token = self.create_user_auth_token(user=self.admin_user, scope_list=["team:admin"])
```

Comment:
this is because the user token `team:admin` scope is `team:admin` on the org level?

Reply:
Org admins have `team:admin`, but they shouldn't be able to act on teams they're not a member of